i18n/en.json

```diff
@@ -106,6 +106,8 @@
   "add_incoming_webhook.url": "<b>URL</b>: {url}",
   "add_incoming_webhook.username": "Username",
   "add_incoming_webhook.username.help": "Choose the username this integration will post as. Usernames can be up to 22 characters, and may contain lowercase letters, numbers and the symbols \"-\", \"_\", and \".\" .",
+  "add_incoming_webhook.channelLocked": "Lock to this channel",
+  "add_incoming_webhook.channelLocked.help": "If selected, the incoming web hook can only post to the selected channel above.",
```

Comment:
Minor wording change, but I think "the channel selected above" sounds a bit better than "the selected channel above"

Reply:
@grundleborg Propose `If set, the incoming webhook can only post to the channel selected above.`